website/docs/sql-glossary/aggregate-functions/sql-round.md

```diff
@@ -1,15 +1,16 @@
 ---
-id: sql-round 
+id: round 
 title: SQL ROUND 
 description: The ROUND aggregate function allows you to round numeric fields or values in SQL to the number of decimal places of your choosing.
-slug: /sql-function-glossary/sql-round
+slug: /sql-glossary/round
 ---
 
 <head>
     <title>Working with SQL ROUND</title>
 </head>
 
-If you’re reading this, that probably means you’re a data person. And as a data person who’s likely modeling data for analytics use cases or analysis, you’re going to need to round data from time to time.
+If you’re reading this, that probably means you’re a data person. And as a data person who’s likely modeling data for analytics use cases or analysis, you’re going to need to round data from time to time. For the unacquainted, "rounding" is making a number simpler so that it's easier to understand while keeping it close to its original value. In data, a common use case for rounding is to decrease the number of decimal places a numeric record has.
```

Comment:
Could leave the "or analysis" off of "modeling data for analytics use cases or analysis"